assets/js/components/ExecutionResults/ExecutionResults.jsx

```diff
@@ -121,9 +122,10 @@ function ExecutionResults({
       <ResultsContainer
         catalogError={false}
         clusterID={clusterID}
-        hasAlreadyChecksResults
+        hasAlreadyChecksResults={!!(executionData || executionLoading)}
```

Comment:
The next line `selectedChecks`, we would need to change it.
This way, you are using current check results, but that field is not for that, it is to get the currently selected checks.
To make it proper, we would need to get the selected checks in the `ExecutionResultsPage`, like `cluster?.selected_checks`, pass them to this view, and use them in that param

Reply:
Done that,  ExecutionResultPage passes me the selected checks to the ExecutionResults